modules/services/kbfs.nix

```diff
@@ -0,0 +1,62 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.kbfs;
+
+in
+
+{
+  options = {
+    services.kbfs = {
+      enable = mkEnableOption "Keybase File System";
+
+      mountPoint = mkOption {
+        type = types.str;
+        default = "keybase";
+        description = ''
+          Mountpoint for the Keybase filesystem, relative to $HOME.
```

Comment:
The option descriptions should be docbook formatted so in this case you can replace `$HOME` by `<envar>HOME</envar>`.

Reply:
Also I think the description should start with "Mount point" to be consistent with the `mountPoint` attribute name.

Reply:
Thanks, I've pushed these changes.